addons/mail/models/mail_tracking_value.py

```diff
@@ -44,6 +44,26 @@ def _compute_field_groups(self):
             field = model._fields.get(tracking.field.name)
             tracking.field_groups = field.groups if field else 'base.group_system'
 
+    def _tracking_value_format(self):
```

Comment:
Define it after create_tracking_values (create_tracking_values is a crud-like method)

While we are at it probably that get_display_value and its 2 siblings could be set as private.

Reply:
The 2 siblings of the method are called in `mail_thread.py`. If we want to make it private then we have to refactor the code in `mail_thread.py` and it should be better to do it in a separate task I think?

Reply:
They are called two times in tool methods that are also private. I don't think we need the overhead of a task and another PR just to change the naming of 3 tool methods used in 2 other tool methods ^^ .